.tests/testScript_noBaseUnits.m

```diff
@@ -7,12 +7,12 @@
 baseUnitSystem = 'none';
 
 %% Normal
-assert(1==u.m)
-assert(0.3048==u.ft)
+assert(1==u.m)  %FIXME: fails with `Incompatible units. Cannot perform operation on variables with different units.` what is an expected behaviour of comparing a number with unit? magnitude comparison?
```

Comment:
Once classes are cleared and line 7 (`baseUnitSystem = 'none';`) is run, the class `u` then contains doubles, not DimVars.

Reply:
Ok. There is a bug setting clas's constant property over `evalin(...)` here

 https://github.com/sky-s/physical-units-for-matlab/blob/bd7f3edf63cfdaf0997612b25a65b00c1c5a38ad/u.m#L72

if running from base workspace it works as expected, but if the caller workspace does not match base (as in case of the test suite), then the class behaiviour will be wrong (appearance of the `baseUnitSystem` in the current workspace will be ignored). So setting a breakpoint during this test reveals the problem:
``` evalin('base','baseUnitSystem')

ans =

  10×2 cell array

    {'m'   }    {[1]}
    {'kg'  }    {[1]}
    {'s'   }    {[1]}
    {'A'   }    {[1]}
    {'K'   }    {[1]}
    {'mol' }    {[1]}
    {'cd'  }    {[1]}
    {'bit' }    {[1]}
    {'¤'   }    {[1]}
    {'unit'}    {[1]}
```
While within the test suite it looks like:
```
K>> evalin('caller','baseUnitSystem')

baseUnitSystem =

    'none'
```

Reply:
fixed with https://github.com/iAnyKey/physical-units-for-matlab/commit/3bb16f89dc39a85a5f0e0396d7056e85042b4035